imports/node-app/plugins/email-smtp/mutations/verifySMTPEmailSettings.test.js

```diff
@@ -1,5 +1,5 @@
-import verifySMTPEmailSettings from "./verifySMTPEmailSettings";
-import mockContext from "/imports/test-utils/helpers/mockContext";
+import verifySMTPEmailSettings from "./verifySMTPEmailSettings.js";
+import mockContext from "/imports/test-utils/helpers/mockContext.js";
```

Comment:
```suggestion
import mockContext from "@reactioncommerce/api-utils/tests/mockContext.js";
```

Reply:
I tested this change, and it works, so I am going to commit it myself for the sake of saving time to get this merged.